### ISSUE TITLE
Open standards: check how we use 'meet' vs 'compliant' in the requirements 

### ISSUE THREAD

**Issue by ElenaFdR** [publiccodenet/standard#386]
This sentence is hard to parse while skimming:
```
* Functionality using features from a non-open standard (one that doesn't meet the [Open Source Initiative Open Standard Requirements](https://opensource.org/osr)) MAY be provided if necessary, but only in addition to compliant features.
```

I think we mean features mandated by an OSI compliant standard.

**Comment by ericherman** [comment]
fixed by 10555ee2abdc0fd21fe4ab97ebc302a53838faf0
